### DISCUSSION TITLE
Server theme mirroring

### DISCUSSION THREAD

**Discussion by 76pfred** [jellyfin/jellyfin-androidtv#1843] [General]
Hi.
Using the server theme customisation plugin, I've modified the server interface to something that I'm really happy with.
The Windows desktop client & Android phone client mirror this theme, but the Android TV client does not.
Is this something that can be looked at in future builds?

**Comment by nielsvanvelzen** [comment]
No

**Reply by 76pfred** [reply]
OK, thanks anyway. I appreciate you taking the time to give such a detailed & comprehensive response.

**Reply by sambartik** [reply]
> OK, thanks anyway. I appreciate you taking the time to give such a detailed & comprehensive response.

As far as I know, the thing is that Windows desktop and Android client UI is built with the same technology jellyfin web is while at the same time having almost the same source code, allowing for the customizations. Whereas Android TV client frontend is built in Java and Kotlin that have totally different technologies and different UI altogether.

**Reply by 76pfred** [reply]
Thank you, sambartik. 
Fair enough, that makes sense :)

**Reply by oddstr13** [reply]
You might be able to use the android (non-tv) client?

**Reply by mueslimak3r** [reply]
Expanding on a prior comment, custom themes (custom CSS) can be shared between web, android (mobile), and maybe a couple others because they are all using what is essentially a web browser to serve the "browsing" parts of the app. Then typically just the video player is implemented natively for the specific platform. These custom themes are made using CSS (styling language for web), and for the most part clients which also run the same frontend code as the web client will be able to use the custom CSS from the themes.

The Android TV app on the other hand doesn't use any web code, and doesn't use CSS. So themes for web are incompatible with native android apps and anything that doesn't use the web code. If someone wanted to create new themes for the android TV app, they would need to be created specifically for android. Here's an example of one of the app's themes "muted purple":
https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/res/values/theme_mutedpurple.xml

**Comment by 76pfred** [comment]
Hi again.
I didn't want to start a new topic to ask this question, as I suspect the answer will be similar to the one above:

Q: Is the JellyfinTV app able to mirror the server's detailed media info/metadata (Atmos/DTS-X & HDR/Dolby Vision)?
